mypy/git.py

```diff
@@ -1,18 +1,28 @@
 """Utilities for verifying git integrity."""
 
+from __future__ import print_function
```

Comment:
This module must now support running in a Python 2.x context so that `python setup.py` in mypy_extensions works.

/mypy_extensions/setup.py --> /setup.py --> mypy/git.py